### ISSUE TITLE
Add a guide for people migrating from airbnb, xo and standard

### ISSUE THREAD

**Issue by alexewerlof** [schibsted/eslint-config-spt#61]
**if** we are going to have differences, we need to facilitate transitioning from other popular configs and motivate the reason why we made a different choice.

**Comment by SimenB** [comment]
I don't think we need to justify changes from standard and xo, but we can point out the main differences. I agree on the airbnb case, though

And transitioning I'd think was mostly just `eslint . --fix`.

**Comment by alexewerlof** [comment]
> I don't think we need to justify changes from standard and xo, but we can point out the main differences.

Deal 👍 

> And transitioning I'd think was mostly just eslint . --fix

Ideally all of our differences is something that `--fix` [can handle](http://eslint.org/docs/rules/).

**Comment by ivarconr** [comment]
close this then?

**Comment by alexewerlof** [comment]
It's WIP. Will close this when next is in good shape.
